game/__init__.py

```diff
@@ -16,8 +18,8 @@ def __init__(self, poll_cb=None):
         #pygame.font.init()
         #pygame.mixer.init(44100)
         ########################
-        self.width = 1200
-        self.height = 900
+        self.width = Gdk.Screen.width()
+        self.height = Gdk.Screen.height()
```

Comment:
Use either pygame.display.Info to get current_h and current_w, or use pygame.display.get_surface and then get_width and get_height.